.gitignore

```diff
@@ -5,5 +5,7 @@
 /node_modules/
 /report/
 /typings/
+/.idea/
+/dist/
 .DS_Store
```

Comment:
technically not part of the issue. 

/,idea/ has already been added in master. Rebase your branch to see the effect :) 
/dist/ will be added by fixing #21 so you may remove it as well.